tools/test-npm.bat

```diff
@@ -0,0 +1,36 @@
+setlocal
+@rem Port of tools/test-npm.sh to windows
+cd %~dp0\..
+
+if not defined NODE_EXE (
+  echo No node executable provided. Bailing 1>&2
+  exit 0
+)
+
+rd /s /q test-npm
+del test-npm.tap
+
+robocopy deps\npm test-npm /e >robocopy.log
+cd test-npm
+
+rd /s /q npm-cache npm-tmp npm-prefix
+mkdir npm-cache npm-tmp npm-prefix
+
+set npm_config_cache=%cd%\npm-cache
+set npm_config_prefix=%cd%\npm-prefix
+set npm_config_tmp=%cd%\npm-tmp
+
+for /f "usebackq tokens=*" %%a in (`..\%NODE_EXE% -p "require('path').resolve('../Release')"`) do set PATH=%%a;%PATH%
+
+node cli.js rebuild
+node cli.js install --ignore-scripts
+
+rem windows doesn't have tee, so not sure how to get output on screen and in file
+if defined debug (
+  node_modules\.bin\tap.cmd --timeout 240 --reporter=tap test\tap
```

Comment:
Is there a reason that the `.bat` directly runs `tap.cmd` while the `.sh` runs `node cli.js run-script test-node`?